test/application_system_test_case.rb

```diff
@@ -18,4 +18,10 @@ def initialize(*args)
       .to_return(:status => 404)
     super(*args)
   end
+
+  # Phantomjs can pick up browser Accept-Language preferences from your desktop environment.
+  # We don't want this to happen during the tests!
+  setup do
+    page.driver.add_headers("Accept-Language" => "en")
+  end
```

Comment:
As much as we may want this I think it at least needs to be in a separate commit... I assume it got added accidentally with something else?

Reply:
It was deliberately in this commit, since the change to the issue comments submit button exposed the problem with the language handling in the system tests. So I thought it was useful to put them both in the same commit to fully illustrate why I was doing it and what sort of test it affects - but I can break it into two commits if you prefer.

Reply:
Ah ok. I hadn't actually even worked out which commit it came from, I was just surprised to see it.